.github/workflows/pr-build.yml

```diff
@@ -0,0 +1,48 @@
+name: PR Build Compose
+
+on:
+  pull_request:
+    paths-ignore:
+      - "**.md"
+      - ".idea/**"
```

Comment:
Odd ignore rules; these should never be committed.

Reply:
They were rules from the old compose repo, seems like since .md and and .idea don't actually change the final build so there is no need to create a test.

Unless you are saying they should have never been commited even in the old repo, in which case I ask why?

Reply:
This would assume these are the only paths that should be ignored. You can use a whitelist to only build PRs if pushed to selected paths.

Reply:
And yes, they should've also not been in the "old" repo.

Reply:
What would the whitelisted paths be?

Reply:
I'd say anything worthwhile to trigger a PR release. Source, for example? With a whitelist you'd also have to consider build files probably, which is why simply triggering on any kind of push is fine I suppose.

Reply:
I am assuming:
```
.github/workflows/pr-build.yml
app/**
assets/**
lib/**
package-lock.json
package.json
pubspec.yaml
```

`fastlane` might also need to be in there as well.

Reply:
I am assuming:
```
.github/workflows/pr-build.yml
app/**
gradle/**
build.graddle.kts
gradle.properties
settings.gradle.kts
```

Reply:
Probably, yeah